src/inference/hmc.jl

```diff
@@ -178,13 +178,22 @@ function DynamicPPL.initialstep(
     # If no initial parameters are provided, resample until the log probability
     # and its gradient are finite.
     if init_params === nothing
+        init_attempt_count = 0
```

Comment:
Wasn't the algorithm above already the first attempt?
```suggestion
        init_attempt_count = 1
```